lib/Cro/Uri/HTTP.pm6

```diff
@@ -1,43 +1,10 @@
 use Cro::Uri :decode-percents, :encode-percents;
 use Cro::HTTP::MultiValue;
 
-class Cro::Uri::HTTP is Cro::Uri {
```

Comment:
That name wants bikeshedding + wants to be in another file

Reply:
Maybe `Cro::ResourceIdentifier::HTTP`.